cli/context/context.go

```diff
@@ -66,8 +66,9 @@ type RunningCtx struct {
 	Daemonize    bool
 	StartTimeout time.Duration
 
-	LogFollow bool
-	LogLines  int
+	LogFollow        bool
+	LogLines         int
+	DisableLogPrefix bool
```

Comment:
What if user specified `start -d --disable-log-prefix`? Should we warn that `--disable-log-prefix` is ignored? What happens now for `start --timeout` (timeout is used for daemonized mode)?